sdcm/cluster.py

```diff
@@ -2084,6 +2084,8 @@ def web_install_scylla(self, scylla_version: Optional[str] = None) -> None:
 
         if match := re.match(r'\D*(\d+\.\d+)', scylla_version or git_branch):
             version = f"nightly-{match.group(1)}"
+        elif git_branch and '/' in git_branch:
```

Comment:
why would a `git_branch` have a `'/'` in it?

Reply:
That's how it just currently works - 
```
09:18:41   > git fetch --tags --progress git@github.com:scylladb/scylla-cluster-tests.git +refs/heads/*:refs/remotes/origin/* # timeout=10
09:18:43  Checking out Revision fac5ac21a6944fb4639777cfe9fef672434dcbbf (origin/master)
09:18:43   > git rev-parse origin/master^{commit} # timeout=10
```

Reply:
good enough as well, fixed.